src/VolumeOfFluidEquationSystem.C

```diff
@@ -487,14 +495,22 @@ VolumeOfFluidEquationSystem::register_initial_condition_fcn(
                                  TurbulenceModel::SST_AMS)
                                   ? true
                                   : false;
+        VectorFieldType* velocity_ = realm_.meta_data().get_field<double>(
+          stk::topology::NODE_RANK, "velocity");
         ScalarFieldType* density_ = realm_.meta_data().get_field<double>(
           stk::topology::NODE_RANK, "density");
-        std::vector<double> userSpec(1);
+        std::vector<double> userSpec(3);
```

Comment:
we could initialize this with an initializer list instead of like this.

Reply:
I agree and changed this over